### PR TITLE
Update use-windows-defender-application-control-policy-to-control-spe…

### PR DESCRIPTION
**Pull request by joinimran**
…cific-plug-ins-add-ins-and-modules.md

Path values in rules were defined incorrectly.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8564

### CODE REVIEW

File: windows/security/threat-protection/windows-defender-application-control/use-windows-defender-application-control-policy-to-control-specific-plug-ins-add-ins-and-modules.md [hunk 1]
```diff
@@ -36,14 +36,14 @@ To work with these options, the typical method is to create a policy that only a
 For example, to create a WDAC policy that allows **addin1.dll** and **addin2.dll** to run in **ERP1.exe**, your organization's enterprise resource planning (ERP) application, run the following commands. Note that in the second command, **+=** is used to add a second rule to the **$rule** variable:
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-application-control/use-windows-defender-application-control-policy-to-control-specific-plug-ins-add-ins-and-modules.md:36]: Suggesting an improvement to the original author's text:
```suggestion
For example, to create a WDAC policy allowing **addin1.dll** and **addin2.dll** to run in **ERP1.exe**, your organization's enterprise resource planning (ERP) application, run the following commands. Note that in the second command, **+=** is used to add a second rule to the **$rule** variable:
```

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]
Will approve but would have preferred the original phrasing.

**Comment by illfated** [comment]
Oh. I see. Fair enough, thank you for sharing your view. I will be less hasty in making such assumptions in the future.

**Review by illfated — APPROVED** [review]

**Review by jsuther1974 — APPROVED** [review]

**Comment by nam31** [comment]
@jsuther1974 PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by illfated** [comment]
@nam31 : As you can see, @jsuther1974 does not have merge rights for this PR, otherwise his approval check mark would have been a full green circle, instead of the regular black one from regular users like JohanFreelancer9 and myself. Not sure if @isbrahm is available for merging, or if this could be done by a team member with actual merge rights. Maybe @denisebmsft has merge access, or possibly @Kateyanne could assist with this too?

**Comment by Kateyanne** [comment]
I have merged this.

Kateyanne

**Pull request merged**